test/vsphere/e2e.yml

```diff
@@ -0,0 +1,125 @@
+name: $PLATFORM_CLUSTER_ID
```

Comment:
@moshloop If I don't make a mistake, we can't interpolate here using env variables as in the other part of the configuration? It is something supported.

Reply:
Yeh - it is currently very manual e.g. https://github.com/flanksource/platform-cli/blob/master/cmd/common.go#L80 

#134 has been logged to support it everywhere

Reply:
I'm going with the following way:

```
platform-cli provision vsphere-cluster -e name="$PLATFORM_CLUSTER_ID" -e domain="${PLATFORM_CLUSTER_ID}.lab.flanksource.com"
```

It should be fine. right?

Reply:
Yes - that should work